src/daemon/config/README.md

```diff
@@ -128,16 +128,21 @@ Please note that your data history will be lost if you have modified `history` p
 
 ### [logs] section options
 
+There are additional configuration options for the logs. For more info, see [Netdata Logging](https://learn.netdata.cloud/docs/developer-and-contributor-corner/libnetdata/log).
```

Comment:
```suggestion
There are additional configuration options for the logs. For more info, see [Netdata Logging](/src/libnetdata/log/README.md).
```

Hey @Steve8291 :wave:! We don't use Learn links, there is a process that parses github ones

Reply:
Thanks @Ancairon